spring-cloud-aws-messaging/src/test/java/org/springframework/cloud/aws/messaging/endpoint/ComplexNotificationEndpointControllerTest.java

```diff
@@ -100,8 +119,8 @@ void notification_notificationReceivedAsMessageWithComplexContent_notificationSu
 				.content(notificationJsonContent)).andExpect(status().isNoContent());
 
 		// Assert
-		assertThat(this.notificationTestController.getMessage().getFirstName()).isEqualTo("Agim");
-		assertThat(this.notificationTestController.getMessage().getLastName()).isEqualTo("Emruli");
+		assertThat(this.notificationTestController.getMessage()).isEqualTo("Agim");
```

Comment:
This is intentional?

Reply:
Ups. Forgot to return back after i did tests to reproduced and error. Fixed